c/CMakeLists.txt

```diff
@@ -1,57 +1,8 @@
-cmake_minimum_required(VERSION 2.8.7)
+cmake_minimum_required(VERSION 3.2)
 project(libsbp)
 
-# Setup flags for Code Coverage build mode
-set(CMAKE_CXX_FLAGS_COVERAGE "${CMAKE_CXX_FLAGS_DEBUG} --coverage" CACHE STRING
```

Comment:
Can we check `git blame` on these chunks of cmake?  Is there potentially something downstream that would use this?

Reply:
`git blame` shows all of these removed lines were added in commit https://github.com/swift-nav/libsbp/commit/062a3e3ec351303023e556a95ee880b6e11e2479 (the same one the removed `.cmake` files were added) and they haven't been modified since. TBH I'm guessing this was all copy pasta from somewhere else.

With how CMake works setting these variables will only have an effect on the stuff defined in this file and any directories it adds via `add_subdirectories()`. These variables aren't used anywhere in the libsbp repo, so removing them shouldn't effect the library itself. Anything downstream that does a `add_subdirectory(libsbp/c/)` also shouldn't see any effects of removing them due to CMake's scoping rules (unless they are doing something really strange with their CMake).

Reply:
Sounds good, thanks.